doc/api/fs.md

```diff
@@ -3947,7 +3947,9 @@ When `file` is a file descriptor, the behavior is similar to calling
 a file descriptor.
 
 The `encoding` option is ignored if `data` is a buffer.
-If `data` is a normal object, it must have an own `toString` function property.
+
+\* If `data` is a normal object, it must have an own (not inherited) `toString`
+function property.
```

Comment:
Avoid `normal object`; my suggestion is not ideal so if you have a better suggestion please go ahead.
```suggestion
\* If `data` is of type `object`, but is not an {ArrayBufferView} and it doesn't
 have an own `toString` property, a `TypeError` is thrown.
```

Reply:
I didn't add "normal object". I think "plain object" is probably good though.

Reply:
Cool, will update in my next commit 🙂